monitoring/thread_status_updater.cc

```diff
@@ -175,31 +195,31 @@ Status ThreadStatusUpdater::GetThreadList(
     // use "memory_order_relaxed" to load the cf_key.
     auto cf_key = thread_data->cf_key.load(std::memory_order_relaxed);
 
-    ThreadStatus::OperationType op_type = ThreadStatus::OP_UNKNOWN;
+    ThreadStatus::OperationType op_type =
+        thread_data->operation_type.load(std::memory_order_acquire);
     ThreadStatus::OperationStage op_stage = ThreadStatus::STAGE_UNKNOWN;
     ThreadStatus::StateType state_type = ThreadStatus::STATE_UNKNOWN;
     uint64_t op_elapsed_micros = 0;
     uint64_t op_props[ThreadStatus::kNumOperationProperties] = {0};
 
-    auto iter = cf_info_map_.find(cf_key);
-    if (iter != cf_info_map_.end()) {
```

Comment:
Honestly I think `SetColumnFamilyInfoKey` mainly serves `cf_info_map_`, just same as relevant book-keeping data structures and functions e.g, `db_key_map_`, `EraseDatabaseInfo` and `EraseColumnFamilyInfo`. They ultimately just skip some thread status info populating to achieve the result of making these thread seem as useless after we call db close or drop cf as in

```
  // A global map that keeps the column family information.  It is stored
  // globally instead of inside DB is to avoid the situation where DB is
  // closing while GetThreadList function already get the pointer to its
  // CopnstantColumnFamilyInfo.
  std::unordered_map<const void*, ConstantColumnFamilyInfo> cf_info_map_;
```
I doubt whether maintaining that much artifacts are needed but don't intend to make optimization this PR just so you know. I only extend into the case where we register main db stress thread where we do non-cf specific operation (db open)